test/README.md

```diff
@@ -2,91 +2,129 @@
+# in the ./edgelet/target/release/ and the ./edgelet/target/hsm/ folders 
+# and named something like aziot-edge*amd64.deb and libiothsm-std*.deb 
+
+./edgelet/build/linux/package.sh
+
+# copy the libiothsm*.deb file into the edgelet release directory, so that the
+# tests can pickup both .deb files from the same folder.
+cp edgelet/target/hsm/libiothsm*.deb edgelet/target/release
+
+~~~
+
+#### Building the container images and pushing them to the local container registry
+From the top folder of the codebase, run the following. 
+~~~ sh
+
+# Build and push images to local container registry
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "edge-hub" -P "Microsoft.Azure.Devices.Edge.Hub.Service"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "edge-agent" -P "Microsoft.Azure.Devices.Edge.Agent.Service"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "load-gen" -P "load-gen"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "direct-method-sender" -P "DirectMethodSender"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "direct-method-receiver" -P "DirectMethodReceiver"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "network-controller" -P "NetworkController"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "relayer" -P "Relayer"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "temperature-filter" -P "TemperatureFilter"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "simulated-temperature-sensor" -P "SimulatedTemperatureSensor"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "test-result-coordinator" -P "TestResultCoordinator"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "number-logger" -P "NumberLogger"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "diagnostics" -P "IotedgeDiagnosticsDotnet"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "metrics-collector" -P "MetricsCollector"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "metrics-validator" -P "MetricsValidator"  -v "latest" --bin-dir target
+sudo scripts/linux/buildImage.sh -r localhost:5000  -i "temperature-filter-function" -P "EdgeHubTriggerCSharp"  -v "latest" --bin-dir target
+
+~~~
+
+## Test
+
+#### Test parameters
+
+The end-to-end tests take several parameters, which they expect to find in a file named `context.json` in the same directory as the test binaries (e.g., `test/Microsoft.Azure.Devices.Edge.Test/bin/Debug/netcoreapp3.1/context.json`). Parameter names are case-insensitive. See [end-to-end test parameters](./doc/end-to-end-test-config.md) for details. 
```

Comment:
On my linux box, for 1.1 - the path to the binaries is test/Microsoft.Azure.Devices.Edge.Test/obj/Debug/netcoreapp3.1/

There's no bin folder for Microsoft.Azure.Devices.Edge.Test/ but there is a bin and an obj for Microsoft.Azure.Devices.Edge.Test.Common/

(It's giving me pause to add this because I'm not sure if I missed a step somewhere or if this is a possible oversight in the documentation.)

Reply:
@gauravIoTEdge I think you want to use the binaries under `bin/` not `obj/`. The bin directory should be there if you ran buildBranch.sh. If you only see `obj/` and not `bin/`, that could happen if you didn't run buildbranch.sh, but Omnisharp (VS Code) built the object files to enable Intellisense.

Reply:
I added a note to address this.